plugins/database/quota/src/org/apache/cloudstack/api/command/QuotaSummaryCmd.java

```diff
@@ -59,7 +59,7 @@ public QuotaSummaryCmd() {
     public void execute() {
         Account caller = CallContext.current().getCallingAccount();
         List<QuotaSummaryResponse> responses;
-        if (caller.getAccountId() <= 2) { //non root admin or system
+        if (caller.getType() == Account.ACCOUNT_TYPE_ADMIN) { //admin account
```

Comment:
This went from valid entries of 1 or 2 to only 1.  This is an intentional change?